packages/e2e-test-utils/src/insertBlock.js

```diff
@@ -45,7 +45,11 @@ async function waitForInserterCloseAndContentFocus() {
  */
 async function insertBlock(searchTerm) {
   await searchForBlock(searchTerm);
-  await expect(page).toClick('button span', { text: searchTerm });
+  const insertButton = await page.waitForXPath(
+    `//button//span[contains(text(), '${searchTerm}')]`
+  );
+  await insertButton.click();
+  await focusSelectedBlock();
```

Comment:
Instead of focusing the block, could we close the inserter manually somehow?

Reply:
Copied from https://github.com/WordPress/gutenberg/blob/8499a38f6d5f0cdfd86c4bf17d42f37232c72125/packages/e2e-test-utils/src/inserter.js#L147-L162

Reply:
oh, in that case, since our versions of `insertBlock` and `waitForInserterCloseAndContentFocus` are now identical to the ones from Gutenberg, we can just use `insertBlock` from `@wordpress/e2e-test-utils` directly instead of copying everything.

Reply:
Sadly we can't as the version of @wordpress/e2e-test-utils is a month out of date. Also 
> It works properly with the minimum version of Gutenberg 9.2.0 or the minimum version of WordPress 5.6.0.

We support, 5.5, so have maintain our own version of these functions. 

https://github.com/WordPress/gutenberg/tree/trunk/packages/e2e-test-utils

Reply:
> Sadly we can't as the version of @wordpress/e2e-test-utils is a month out of date.

Is it? At first glance I don't see a difference from the package published on npm that we use and the Gutenberg `trunk` branch.

I don't mind maintaining our own version of certain functions if they differ in logic, but here they seem identical to me. But happy to be corrected.

Reply:
This [packages](https://www.npmjs.com/package/@wordpress/e2e-test-utils) says, last published 1 month ago. 

<img width="451" alt="Screenshot 2021-07-07 at 12 22 52" src="https://user-images.githubusercontent.com/237508/124751077-23594d00-df1e-11eb-9fe5-89e51b869db0.png">

Inserter was updated 12 minutes ago. 

<img width="1320" alt="Screenshot 2021-07-07 at 12 22 45" src="https://user-images.githubusercontent.com/237508/124751129-2fdda580-df1e-11eb-8f3c-021e03992122.png">

What am I missing?

Reply:
That seems fine to me? Yes there's a recently modified file in there, but I don't see how that would affect us. [That change](https://github.com/WordPress/gutenberg/pull/32935) is neither in core nor in the package version that we use. We also don't use any of the [currently unreleased](https://github.com/WordPress/gutenberg/blob/f3f532015d3200f8b5b84dc2978e423b72c1fa98/packages/e2e-test-utils/CHANGELOG.md#unreleased) new features.

----

Putting this another way: if we just remove our own version of `insertBlock` and use the version from `@wordpress/e2e-test-utils`, do the e2e tests still pass? If yes, I'd say we can safely remove our copy.